src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,143 @@
+
+## Get ActiveMQ metrics in a dashboard [#send]
+
+Our ActiveMQ dashboard takes data scraped from your JMX integration and translates it into a collection of premade graphs and charts. Housing all your data in the New Relic Platform gives you a full overview of your system's health and critical business points. 
+
+<img
+  title="Access_dashboards.png"
+  alt="A screenshot that identifies where to find the Dashboards tab in the top nav."
+  src={accessDashboards}
+/>
+<figcaption>
+**[one.newrelic.com](https://one.newrelic.com)** > Hover to the top navigation > Click **Dashboards**: After installation, you can find your dashboards in the New Relic platform. 
+</figcaption> 
+
+<CollapserGroup>
+    <Collapser
+    id="step1"
+    title="Step 1. Update the `jmx-config.yml` file"
+  >
+The below YAML snippet connects your ActiveMQ metrics via `CONNECTION_URL` to jconsole, a Java tool that makes those metrics readable. If you're following along with the JMX integration install, this step corresponds with the configuration step. 
+
+To update: 
+
+1. Go to file path: `/etc/newrelic-infra/integrations.d/jmx-config.yml`
+2. Add this snippet:
+
+```yml
+integrations:
+- name: nri-jmx
+  env:
+    COLLECTION_FILES: "/etc/newrelic-infra/integrations.d/jmx-config.yml"
+    JMX_HOST: 127.0.0.1
+    JMX_PASS: admin
+    JMX_PORT: 8161
+    JMX_user: admin
+    CONNECTION_URL: "service:jmx:rmi://127.0.0.1:1098/jndi/rmi://127.0.0.1:1099/jmxrmi"
+    REMOTE_MONITORING: "true"
+  interval: 15s
+  labels:
+    env: staging
+```
+  </Collapser>
+  <Collapser
+    id="step2"
+    title="Step 2. Update the `jvm-metrics` with ActiveMQ metric names"
+  >
+This step organizes your metrics to be human readable once inside the New Relic platform. To update `jvm metrics`:
+
+1. Go to file path `/etc/newrelic-infra/integrations.d/jvm-metrics.yml`
+2. Add this snippet:
+
+```yml
+collect:
+  - domain: java.lang
+  event_type: JVMSampleActiveMQ
+  beans:
+    - query: type=GarbageCollector,name=*
+      attributes:
+        - CollectionCount
+        - CollectionTime
+    -query: type=Memory
+      attributes:
+        - HeapMemoryUsage.Committed
+        - HeapMemoryUsage.Init
+        - HeapMemoryUsage.Max
+        - HeapMemoryUsage.Used
+        - NonHeapMemoryUsage.Committed
+        - NonHeapMemoryUsage.Init
+        - NonHeapMemoryUsage.Max
+        - NonHeapMemoryUsage.Used
+    - query: type=Threading
+      attributes:
+        - ThreadCount
+        - TotalStartedThreadCount
+    - query: type=ClassLoading
+      attributes:
+        - LoadedClassCount
+    - query: type=Compilation
+      attributes:
+        - TotalCompilationTime
+```
+  </Collapser>
+  <Collapser
+    id="step3"
+    title="Step 3. Forward your ActiveMQ logs"
+  >
+With your monitoring environment set up, forward your ActiveMQ logs to the New Relic Platform. To forward logs:
```

Comment:
@zuluecho9 Do you think this step 3 should be removed too to avoid redundancy? i.e., linking to the forward your logs doc instead of dedicating a step to it?